templates/revoke.sh.j2

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+
+CADIR="{{ openvpn_key_dir }}"
+
+cd ${CADIR}
+
+gen_crl () {
+  # regenerate the certificate revocation list
+  openssl ca -gencrl -config ca.conf -out ca-crl.pem
```

Comment:
If these are single line commands, could you do it in ansible directly instead of a shell script?